AIPs/aip-5.md

```diff
@@ -1,212 +1,176 @@
-The current Ampleforth rebasing function predisposes the network to short periods of rapid expansion and long periods of gradual contraction. This document proposes an update to the Ampleforth rebasing policy that would:
+This document proposes an update to the Ampleforth rebase policy that would limit the protocol's sensitivity to short-lived demand shocks.
 
-* Limit the protocol's sensitivity to short-lived demand shocks.
-* Create symmetry between supply expansion and contraction.
+This should make the Ampleforth protocol a more robust building block for decentralized finance.
 
-The two above considerations should make the Ampleforth protocol a more balanced and robust building block for decentralized finance.
 
 ## Abstract
-<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the AIP is implemented, not *why* it should be done or *how* it will be done. If the AIP proposes deploying a new contract, write, "we propose to deploy a new contract that will do x".-->
-We propose to deploy a new contract that updates the current supply policy's linear rebasing function with a modified sigmoid-like rebasing function. 
+
+The proposal is to deploy a new contract that updates the current supply policy's linear rebase function with a sigmoid (s-shaped) function.
+
 
 ## Motivation
-<!--This is the problem statement. This is the *why* of the AIP. It should clearly explain *why* the current state of the protocol is inadequate.  It is critical that you explain *why* the change is needed, if the AIP proposes changing how something is calculated, you must address *why* the current calculation is innaccurate or wrong. This is not the place to describe how the AIP will address the issue!-->
 
-The original Ampleforth linear rebasing function was intended to make minimal assumptions about how price reacts to expansion and contraction (i.e. expansion would slow price increases by translating them into supply increases and contraction would slow price decreases by translating them into supply decreases). Said differently, the main purpose of the function is to point supply changes in the right direction without trying to predict the exact magnitude of supply change needed for the fastest reversion to the price target.
+The original Ampleforth linear rebase function was intended to make minimal assumptions about how price reacts to expansion and contraction--i.e. expansion would slow price increases by translating them into supply increases and contraction would slow price decreases by translating them into supply decreases. Hence the main purpose of the function is to point supply change in the right direction without trying to predict the magnitude of supply change needed for the fastest convergence to the price target.
```

Comment:
1) Suggest using an mdash — rather than two small dashes
2) Suggest the following language: "Hence the main purpose of the function is to point supply changes in the right direction without trying to predict the magnitude of change needed for the fastest convergence to the price target"